.github/ISSUE_TEMPLATE/new_package.yml

```diff
@@ -69,34 +69,35 @@ body:
       label: Category
       description: Which category should this tool be installed to?
       options:
+        - Active Directory
         - Android
         - Cloud
+        - Command & Control
         - Debuggers
         - Delphi
         - Disassemblers
-        - dotNet
+        - Evasion
+        - Exploitation
         - Forensic
         - Hex Editors
+        - Information Gathering
+        - InnoSetup
         - Java
         - Javascript
         - Networking
         - Office
         - PDF
         - PE
+        - Password Attacks
         - PowerShell
         - Python
         - Text Editors
         - Utilities
         - VB
-        - Active Directory
```

Comment:
I kind of liked that we separated the FLARE VM and Commando VM categories. Although separate lists would probably make more sense.

Reply:
This is displayed to the user. I was thinking alphabetically would make it easier for users. I don't see how we could have separate lists here, can you elaborate?